shopBuddy/shopBuddy/Base.lproj/Main.storyboard

```diff
@@ -273,6 +290,58 @@
+                                <rect key="frame" x="0.0" y="96" width="414" height="652"/>
+                                <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMaxY="YES"/>
+                                <color key="backgroundColor" systemColor="labelColor"/>
+                            </view>
+                            <toolbar opaque="NO" clearsContextBeforeDrawing="NO" contentMode="scaleToFill" fixedFrame="YES" translatesAutoresizingMaskIntoConstraints="NO" id="10r-L6-BoQ">
+                                <rect key="frame" x="0.0" y="748" width="414" height="44"/>
+                                <autoresizingMask key="autoresizingMask" widthSizable="YES" flexibleMinY="YES"/>
+                                <items>
+                                    <barButtonItem width="141" style="plain" systemItem="fixedSpace" id="vr2-Yt-i26"/>
+                                    <barButtonItem title="Start Scanning" id="sWu-N3-Kej">
+                                        <connections>
+                                            <action selector="startStopScanning:" destination="CGZ-BZ-W37" id="Vzk-rU-189"/>
+                                        </connections>
+                                    </barButtonItem>
+                                    <barButtonItem style="plain" systemItem="fixedSpace" id="POP-uT-1EC"/>
+                                </items>
+                            </toolbar>
+                            <button opaque="NO" contentMode="scaleToFill" fixedFrame="YES" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="system" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="uaD-oR-vX1">
+                                <rect key="frame" x="0.0" y="0.0" width="414" height="88"/>
+                                <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMaxY="YES"/>
+                                <state key="normal" title="Button"/>
+                                <buttonConfiguration key="configuration" style="plain" title="Lookup"/>
+                                <connections>
+                                    <action selector="didTapLookup:" destination="CGZ-BZ-W37" eventType="touchUpInside" id="CLW-Ti-NPH"/>
+                                </connections>
+                            </button>
+                        </subviews>
+                        <viewLayoutGuide key="safeArea" id="6wz-tm-CMP"/>
+                        <color key="backgroundColor" systemColor="systemBackgroundColor"/>
+                    </view>
+                    <toolbarItems/>
+                    <simulatedToolbarMetrics key="simulatedBottomBarMetrics"/>
+                    <connections>
+                        <outlet property="lookupButton" destination="uaD-oR-vX1" id="RLG-tY-aZz"/>
```

Comment:
I don't think you plan to keep this in your final design right? I mean you are not showing the barcode number in the UI, right?

Reply:
Yeah, when I implement the timer, I will eliminate the barcode number in the UI. This is mostly just for us to confirm whether or not I want to call the API or not (to save queries before I have to switch keys).